src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -117,6 +113,9 @@ def __init__(self, name: str, address: str):
         >>> qcm_module = QcmBb(name="qcm_bb", address="192.168.1.100:2", cluster=cluster_instance)
         """
         super().__init__(name, address)
+        DEFAULT_SEQUENCERS = {"o1": 0, "o2": 1, "o3": 2, "o4": 3}
+        FREQUENCY_LIMIT = 500e6
+        OUT_PORT_PATH = {0: "I", 1: "Q", 2: "I", 3: "Q"}
```

Comment:
Just noting that this is different behavior than before and now these are completely ignored (local variables in `__init__` that are not used anywhere?). Probably doesn't matter since qblox is broken anyway and we may want to keep here for when we rewrite.

Reply:
Yes: since the transition to Pydantic, these were breaking top-level because of the missing type hint.
I didn't want to lose completely the information, and I moved inside to avoid errors during linting/testing.

However, this is because I also assumed that Qblox is broken, and it will be rewritten. And, to be fair, the information is preserved in https://github.com/qiboteam/qibolab/tree/0.1. We could even decide to just delete all broken drivers.